### PR TITLE
Dragdrop

### PR DESCRIPTION
**Pull request by mkirvan00**

### CODE REVIEW

**Comment by bhujoshi** [comment]
@mkirvan00 Thanks for spending time do work on this feature. I have checked it and drag and drop working pretty fine but I can see a few gaps in the current implementation and I feel that is completely my fault because the ticket did not contain enough information. I will add the same on the ticket as well.
react-todo is an npm package. It is open-source and inspired by [keep](https://keep.google.com/) to-do list. We want to keep UI similar to [keep](https://keep.google.com). The drag icon should appear on the hover of the item. Like below image 
drag-and-drop-example

Also, we want to minimize peer dependency so that more people are able to use it. like you have added `styled-components` and `react-beautiful-dnd`. If possible do it without `react-beautiful-dnd` but `styled-components` is not needed. And add dependencies inside peer dependencies in packge.json if using any.

File: src/components/Todo/dist/Todo.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
+"use strict";
```
Comment (bhujoshi) [inline src/components/Todo/dist/Todo.js:1]: Do not push anything from the dist folder. I am adding dist folder inside .gitignore please that a pull

**Comment by bhujoshi** [comment]
Let me know in case any help is needed from me. Thanks

**Comment by kamalkishor1991** [comment]
Closing this because of inactivity. @mkirvan00 please feel free to reopen if you plan to work on this.

**Pull request closed without merging**